webpack/readme.md

```diff
@@ -0,0 +1,39 @@
+# Webpack UIDS build
```

Comment:
If we are going to add something like this, I would like to see how we could do it for the entire library. I would like to discuss this idea some more.